server/routes/shared_api/observation/point.py

```diff
@@ -73,6 +74,9 @@ def point_within():
   if not variables:
     return 'error: must provide a `variables` field', 400
   date = request.args.get('date') or DATE_LATEST
+  # If querying by highest coverage is disabled, default to DATE_LATEST
+  if date == DATE_HIGHEST_COVERAGE and not is_highest_coverage_enabled():
```

Comment:
Consider adding a TODO to remove this once b/336358566 is fixed.

Reply:
good call. added